docs/usage/config-presets.md

```diff
@@ -16,20 +16,22 @@ In short:
 
 ## Goals of Preset Configs
 
-The main reason for supporting preset configs is to decrease duplication:
+The main reason for supporting preset configs is to decrease duplication.
+By using a preset config you:
 
-1.  You shouldn't need copy/paste the same config across all your repositories
-2.  You shouldn't need to reinvent any config "wheels" that others have invented before
+1. Avoid duplicating the same config across all your repositories
+2. Can use someone else's configuration and extend it
 
-A further reason was to make Renovate configuration "self-documenting", by adding the `"description"` field to all preset configs.
+Renovate's configuration is self-documenting, because you can fill in the `"description"` field in all preset configs.
 
 ## Implementation Approach
 
-In order to achieve the above goals, preset configs have been implemented to allow a very modular approach - preset configs may be as small as a partial package rule or as extensive as an entire configuration, like an `eslint` config.
+In order to achieve these goals, preset configs allow for a very modular approach - preset configs can be as small as a partial package rule or as large as an entire configuration, like an `eslint` config.
 
 ## Preset Hosting
 
-Presets can be defined using either npm packages, or with GitHub/GitLab repositories. Bitbucket-hosted presets are yet to be implemented.
+Presets can be defined using either npm packages, or with GitHub/GitLab repositories.
+Bitbucket-hosted presets are not supported for now.
```

Comment:
I would maybe say that something like "presets hosted elsewhere are not supported." Or maybe just remove this sentence entirely?

Reply:
```suggestion
Presets that are hosted elsewhere are not supported.
```